### PR TITLE
Adopt copier template

### PR DESCRIPTION
**Pull request by AlexanderWells-diamond**
A fair amount of manual work went in to fixing the Docs, to make the existing .rst work with the new .md format. The new documentation should be reviewed to check it is sensible.

### CODE REVIEW

**Comment by AlexanderWells-diamond** [comment]
@coretl The remaining errors are a handful of new linting errors, and a single test failure. 

The linting errors are mostly regarding accessing private members, which would require actual code changes to fix. 

The one test failure is strange - it's completely repeatable, but only after the copier template has been applied. I can't see any obvious packages that would cause it (I tried rolling back `epicscorelibs` and various `pytest-*` dependencies). [It passed here a few months ago](https://github.com/DiamondLightSource/aioca/actions/runs/9990760182/job/27612111584). 

How do you want me to handle these issues?

**Comment by AlexanderWells-diamond** [comment]
After discussion with @coretl I mostly ignored the linting errors, and then spent some time investigating the test failure and applying a (hacky) fix to it. This is now ready for review.

File: .copier-answers.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Changes here will be overwritten by Copier
+_commit: 2.3.0
+_src_path: gh:DiamondLightSource/python-copier-template
+author_email: tom.cobb@diamond.ac.uk
+author_name: Tom Cobb
+component_lifecycle: production
+component_owner: coretl:tmc43
```
Comment (coretl) [inline .copier-answers.yml:7]:
```suggestion
component_owner: group:default/accelerator-controls
```

File: catalog-info.yaml [hunk 2]
```diff
@@ -13,4 +13,4 @@ metadata:
 spec:
   type: library
   lifecycle: production
-  owner: group:Controls
\ No newline at end of file
+  owner: group:Controls
```
Comment (coretl) [inline catalog-info.yaml:16]:
```suggestion
  owner: group:default/accelerator-controls
```

File: src/aioca/__main__.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+from . import __version__
+
+__all__ = ["main"]
+
+
+def main(args=None):
+    parser = ArgumentParser()
+    parser.add_argument("-v", "--version", action="version", version=__version__)
+    args = parser.parse_args(args)
```
Comment (coretl) [inline src/aioca/__main__.py:11]: If you keep the `test_cli.py` that copier made then this will be covered

Comment (AlexanderWells-diamond) [reply src/aioca/__main__.py:11]: Added in https://github.com/DiamondLightSource/aioca/pull/52/commits/afd88d5602d6d944cafef0450d1577bbd6e335fd

**Review by coretl — APPROVED** [review]

**Pull request merged**